functions/go/enable-gcp-services/gcpservices/services.go

```diff
@@ -0,0 +1,55 @@
+package gcpservices
+
+import (
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+// Service represents a valid GCP Service
+type Service struct {
+	APIVersion string `yaml:"apiVersion"`
```

Comment:
You can use https://pkg.go.dev/sigs.k8s.io/kustomize/kyaml@v0.11.1/yaml#ResourceMeta

Reply:
done